src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -330,6 +330,8 @@ public static class Filter
          public static readonly string MARKDOWN_FILE_FILTER = FileFilter("Markdown", MARKDOWN_EXTENSION);
          public static readonly string JSON_FILTER = filter(JSON_EXTENSION);
          public static readonly string XML_FILTER = filter(XML_EXTENSION);
+         public static readonly string PNG_SAVE_FILE_FILTER = "PNG file (*.png)|*.png";
```

Comment:
what about you use the method that is used throughout `FileFilter` ? consistency :)